src/sentry/discover/endpoints/serializers.py

```diff
@@ -163,9 +163,10 @@ class DiscoverSavedQuerySerializer(serializers.Serializer):
     widths = ListField(child=serializers.CharField(), required=False, allow_null=True)
     yAxis = ListField(child=serializers.CharField(), required=False, allow_null=True)
     display = serializers.CharField(required=False, allow_null=True)
+    topEvents = serializers.CharField(required=False, allow_null=True)
```

Comment:
This is just to allow a variable number of top events right? should this be an integer field?